Cyberpunk DBM/readme.md

```diff
@@ -0,0 +1,26 @@
+# CyberPunk DBM
++ Made by borgar#2819
++ Based off on the "Lo Fi Vibe" Theme.
++ I do not own any pictures or gifs in this theme. (The background gif is made by @AffinityPlay on Twitter!)
```

Comment:
`I do __NOT__ own any pictures or gifs in this theme. (The background gif is made by @AffinityPlay on Twitter!)`